Classes/Console/Command/Upgrade/UpgradeRunCommand.php

```diff
@@ -51,17 +52,23 @@ protected function configure()
 
   <code>%command.full_name% all</code>
 
-  <code>%command.full_name% all --no-interaction --confirm all --deny typo3DbLegacyExtension --deny funcExtension</code>
-
-  <code>%command.full_name% all --no-interaction --deny all</code>
+  <code>%command.full_name% all --confirm all</code>
 
   <code>%command.full_name% argon2iPasswordHashes --confirm all</code>
 
+  <code>%command.full_name% all --confirm all --deny typo3DbLegacyExtension --deny funcExtension</code>
```

Comment:
Is there a reason for not having `--no-interaction` here?

Reply:
no, just to make the examples a bit more diverse